src/UKHO.S100PermitService.Common/Services/IUserPermitService.cs

```diff
@@ -1,10 +1,11 @@
-﻿using UKHO.S100PermitService.Common.Models.UserPermitService;
+﻿using System.Net;
+using UKHO.S100PermitService.Common.Models.UserPermitService;
 
 namespace UKHO.S100PermitService.Common.Services
 {
     public interface IUserPermitService
     {
-        Task<UserPermitServiceResponse> GetUserPermitAsync(int licenceId, CancellationToken cancellationToken, string correlationId);
+        Task<(HttpStatusCode httpStatusCode, UserPermitServiceResponse? userPermitServiceResponse)> GetUserPermitAsync(int licenceId, CancellationToken cancellationToken, string correlationId);
```

Comment:
UserPermitServiceResponse required to be nullable?

Reply:
If UserPermitServiceResponse is not marked as nullable then it shows warning as null is returned in case of 404 response. Thanks